frontend/src/Components/RolesAndRules.jsx

```diff
@@ -1,116 +1,160 @@
-            winCondition: 'Get voted out by Town or killed by Mafia.',
-        },
-    ];
-
-    const rulesText =
-        'During Day, the actions of the previous night get revealed. Then, the Town must choose someone to Eliminate;' +
-        ' they are trying to get rid of the Mafia, but the Mafia can lead them astray by casting suspicion elsewhere.' +
-        ' When a player gets a majority of the votes, they are eliminated. and it becomes Night.' +
-        '\n\nDuring Night, the roles get to perform their actions. Mafia members can decide which Town member to kill.' +
-        ' Medics can try to guess who will be killed and save them. Detectives can find out if a Town member is Mafia or not.' +
-        ' Once all actions have been completed, the night will be over.' +
-        '\n\nThe Day and Night cycle repeats until the Town or Mafia win the game.' +
-        'If the Jester ever dies, they win and the game ends.';
-
-    return (
-        <div>
-            <div className={styles.roleContainer}>
-                {roles.map((role) => (
-                    <div className={styles.roleInfo} key={role.name}>
-                        <div>
-                            {role.name.toLowerCase() === userRole.toLowerCase() && (
-                                <p className={styles.roleHighlight}>This is your role!</p>
-                            )}
-                        </div>
-                        <Paper
-                            elevation={2}
-                            className={`${classes.root} ${
-                                role.name === 'Jester' ? classes.jester : role.isGood ? classes.good : classes.bad
-                            }`}
-                        >
-                            <h3>{role.name}</h3>
-                            <p>{role.description}</p>
-                            <div>
-                                <strong>Win Condition:</strong>
-                                <p>{role.winCondition}</p>
-                            </div>
-                        </Paper>
-                    </div>
-                ))}
-            </div>
-            <div>
-                <div className={styles.ruleContainer}>
-                    <p>{rulesText}</p>
-                </div>
-            </div>
-        </div>
-    );
-};
-
-export default RolesAndRules;
+import { React, useContext } from 'react';
+import { Paper } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import styles from '../Styles/RolesAndRules.module.css';
+import { GameContext, GeneralContext } from '../Context';
+import civilianImage from '../images/Civilian.svg';
+import medicImage from '../images/Medic.svg';
+import mafiaImage from '../images/Mafia.svg';
+import jesterImage from '../images/Jester.svg';
+import detectiveImage from '../images/Detective.svg';
+import sun from '../images/Sun.svg';
+import moon from '../images/Moon.svg';
+import yourRole from '../images/Your Role.svg'

```

Comment:
Nit: Probably better to have file names without spaces in them, but shouldn't be an issue here